### PR TITLE
Add zynq docs

### PR DESCRIPTION
**Pull request by acomodi**
This PR adds docs for the zybo board.

They include:

- example for the counter on zybo
- SD card preparation for the zynq ARM core booting process
- Bitstream loading through U-boot

This PR also contains a small fix to the readme collector script that enables to have special tags in the examples READMEs. For instance external references, as well as text modification (italic, bold) were not allowed.

### CODE REVIEW

**Comment by acomodi** [comment]
Docs generated by this PR: https://symbiflow-examples--102.org.readthedocs.build/en/102/

File: docs/collect_readmes.py [hunk 1]
```diff
@@ -168,8 +169,29 @@ def handle_default(block):
...
     ret = {}
+
+
+    final_text = ""
+    if block.tagname.strip() == "paragraph":
+
+        skip_next = False
+        for node in block.traverse():
+            if skip_next:
+                skip_next = False
```
Comment (rw1nkler) [inline docs/collect_readmes.py:180]: When this situation can happen? You are setting `skip_next` to `False` two lines above.

**Review by rw1nkler — COMMENTED** [review]
I have found that [[THIS]](https://github.com/SymbiFlow/symbiflow-examples/pull/102/files#diff-a82f42e59f1c59752ce7669df2bc7ab3094a131348921b24eb679ed311634209R44) section is not displayed correctly in the documentation:
https://symbiflow-examples--102.org.readthedocs.build/en/102/building-examples.html#counter-test
```
For Zybo, please follow the guide on how to load a bitstream from U-boot.
Zybo
guide on how to load a bitstream from U-boot
```
the last two lines are unwanted. 
I think that any fixes to the jinja templates should be added as a separate PR.

**Comment by tmichalak** [comment]
@acomodi I also think that we want to have the fixes to the jinja templates as a separate PR

**Review by tmichalak — CHANGES_REQUESTED** [review]
Please rebase on the current master

File: docs/collect_readmes.py (hunk 1 above)
Comment (acomodi) [reply docs/collect_readmes.py]: Fixed. Removed this and rebased on top of master which has enhanced parsing of inline nodes

File: docs/running-examples.rst [hunk 2]
```diff
@@ -121,3 +138,92 @@ Add IPv4 address to you interface:
...
+
+      cp uboot-linux-images/boot/* /path/to/mountpoint/boot/
+      sync
+
+#. Copy rootfs
+
+   .. code-block:: bash
+
+      wget -qO- http://de5.mirror.archlinuxarm.org/os/ArchLinuxARM-armv7-latest.tar.gz | sudo tar -xvzC /path/to/mountpoint/root
+      sudo cp -a uboot-linux-images/rootfs/* /path/to/mountpoint/root/root/
```
Comment (tmichalak) [inline docs/running-examples.rst:175]: is the destination path correct? I mean above you have `/path/to/mountpoint/boot/` and for rootfs `/path/to/mountpoint/root/root/` - seems one level too deep

Comment (acomodi) [reply docs/running-examples.rst:175]: Yes, the first root is the root file system, while the second one is the root directory within the root filesystem.

E.g. the first `root` contains:
```
bin  boot  lost+found  media  mnt  nix  opt  proc  root ...
```

**Pull request merged**